docs/source/models/risk_effects/hemoglobin/index.rst

```diff
@@ -73,108 +63,166 @@ GBD 2019 Modeling Strategy
      - Outcome ID
      - Affected measure
      - Note
-   * -
-     -
-     -
-     -
-     -
+   * - Maternal hemorrhage
+     - cause
+     - 367
+     - incidence (affects mortality and morbidity in GBD)
+     - In GBD shared functions, but expected to be updated
+   * - Maternal sepsis and other infections
+     - cause
+     - 368
+     - incidence (affects mortality and morbidity in GBD)
+     - In GBD shared functions, but expected to be updated
+   * - Maternal hypertensive disorders
```

Comment:
Not directly relevant to this documentation, but hypertensive disorders, pre-eclampsia, etc. were scoped for Wave 3 of the model. I wonder if it is better to add this later when we tackle blood pressure and related issues more broadly?